test/render-test.js

```diff
@@ -5,7 +5,11 @@ var path = require('path');
 var _files = path.join(__dirname, '_files');
 
 function getContents(testName, ext) {
-  return fs.readFileSync(path.join(_files, testName + '.' + ext), 'utf8');
+  try {
+    return fs.readFileSync(path.join(_files, testName + '.' + ext), 'utf8');
+  } catch (ex) {
+    return null;
+  }
```

Comment:
Included a small fix for running render tests which doesnt exist (e.g. misspelled), instead of some verbose lowlevel fs error.

Results in something like
```
Error: Cannot find view for test "mytest"
   ... <more stacktrace>
```
rather than
```
fs.js:439
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory ....
```